src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/AbstractClassCouplingCheck.java

```diff
@@ -319,6 +326,16 @@ public void visitLiteralNew(DetailAST ast) {
         }
 
         /**
+         * Visits ANNOTATION token for the current class context.
+         * @param ast NEW token.
+         */
+        public void visitAnnotation(DetailAST ast) {
+            if (ast.getParent().getParent().getType() != TokenTypes.ANNOTATION_DEF) {
```

Comment:
Why do we ignore annotations under `ANNOTATION_DEF`?

Reply:
I thought, what need ignore annotation for `@interface`. But it was wrong. I fixed it.